pkg/kubelet/cm/container_manager_linux_test.go

```diff
@@ -165,3 +171,69 @@ func TestSoftRequirementsValidationSuccess(t *testing.T) {
 	assert.NoError(t, err)
 	assert.True(t, f.cpuHardcapping, "cpu hardcapping is expected to be enabled")
 }
+
+func TestGetCapacity(t *testing.T) {
+	ephemeralStorageFromCapacity := int64(2000)
+	ephemeralStorageFromCadvisor := int64(8000)
```

Comment:
My main remaining concern is these two values being wildly different, and the possible knock-on effects it could have on the node. But, from what I've seen they're both retrieved via cadvisor in the exact same way, so barring some wacky cadvisor issue they should be the same quantity.